torchao/quantization/prototype/mixed_precision/scripts/mp_quant_eval.py

```diff
@@ -0,0 +1,125 @@
+import torch
+import torch.nn as nn
+
+from naive_intNwo import intN_weight_only_asym, intN_weight_only_sym
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from lm_eval.models.huggingface import HFLM
+from lm_eval.evaluator import evaluate
+from lm_eval.tasks import get_task_dict
+
+from torchao.quantization import quantize_, int8_weight_only, int4_weight_only, int8_dynamic_activation_int4_weight
+from torchao._models._eval import TransformerEvalWrapper
+
+from torchao.quantization.quant_primitives import (
+    MappingType,
+    ZeroPointDomain,
+)
+
+from torchao.quantization.quant_api import autoquant
+
+
+torch._inductor.config.force_fuse_int_mm_with_mul = True
+torch._inductor.config.fx_graph_cache = True
+
+
+def run_evaluation(repo_id, tasks, limit, device, precision, quantization, compile, batch_size, max_length, sensi_bit, non_sensi_bit, quant_sym, group_size):
+
+    tokenizer = AutoTokenizer.from_pretrained(repo_id)
+    model = AutoModelForCausalLM.from_pretrained(repo_id).to(device="cpu", dtype=precision)
+
+    if quantization == "int8dq":
+        quantize_(model.to(device=device), int8_dynamic_activation_int4_weight())
```

Comment:
this seems wrong? On main it's `int8_dynamic_activation_int8_weight`: https://github.com/pytorch/ao/blob/0e6c1224c6645426ef5dbb1817b2966a9c68ac84/scripts/hf_eval.py#L52.

Actually we can probably just delete this case?

Reply:
removed this for now since we will not use this